apps/crn-server/test/data-providers/reminders.data-provider.test.ts

```diff
@@ -565,93 +570,124 @@ describe('Reminder Data Provider', () => {
+          // set the current date to more than 24 hours after the material updated at
+          const time = DateTime.fromISO(materialUpdatedAt)
+            .plus({ hours: 24, minutes: 1 })
+            .toJSDate();
+          jest.setSystemTime(time);
+
+          const squidexGraphqlResponse = getSquidexRemindersGraphqlResponse();
+          squidexGraphqlResponse.queryResearchOutputsContents = [];
+          squidexGraphqlResponse.queryEventsContents![0]!.flatData[
+            materialUpdatedAtName
+          ] = materialUpdatedAt;
+          squidexGraphqlClientMock.request.mockResolvedValueOnce(
+            squidexGraphqlResponse,
+          );
 
-        const videoRecordingUpdatedAt = '2022-09-01T08:00:00Z';
-        // set the current date to more than 24 hours after the video recording updated at
-        const time = DateTime.fromISO(videoRecordingUpdatedAt)
-          .minus({ minutes: 1 })
-          .toJSDate();
-        jest.setSystemTime(time);
+          const result = await reminderDataProvider.fetch(
+            fetchRemindersOptions,
+          );
+          expect(result).toEqual({
+            total: 0,
+            items: [],
+          });
+        });
 
-        const squidexGraphqlResponse = getSquidexRemindersGraphqlResponse();
-        squidexGraphqlResponse.queryResearchOutputsContents = [];
-        squidexGraphqlResponse.queryEventsContents![0]!.flatData.videoRecordingUpdatedAt =
-          videoRecordingUpdatedAt;
-        squidexGraphqlClientMock.request.mockResolvedValueOnce(
-          squidexGraphqlResponse,
-        );
+        test(`Should not fetch the reminder if a ${material} in an event was updated after the current time`, async () => {
+          /* This in theory could never happen, it would mean that the material will be updated in the future */
+
+          const materialUpdatedAt = '2022-09-01T08:00:00Z';
+          // set current time to 1 minute before the material is going to be updated
+          const time = DateTime.fromISO(materialUpdatedAt)
+            .minus({ minutes: 1 })
+            .toJSDate();
+          jest.setSystemTime(time);
+
+          const squidexGraphqlResponse = getSquidexRemindersGraphqlResponse();
+          squidexGraphqlResponse.queryResearchOutputsContents = [];
+          squidexGraphqlResponse.queryEventsContents![0]!.flatData[
+            materialUpdatedAtName
+          ] = materialUpdatedAt;
+          squidexGraphqlClientMock.request.mockResolvedValueOnce(
+            squidexGraphqlResponse,
+          );
 
-        const result = await reminderDataProvider.fetch(fetchRemindersOptions);
-        expect(result).toEqual({
-          total: 0,
-          items: [],
+          const result = await reminderDataProvider.fetch(
+            fetchRemindersOptions,
+          );
+          expect(result).toEqual({
+            total: 0,
+            items: [],
+          });
         });
-      });
-    });
+      },
+    );
```

Comment:
What if both video and presentation material types are updated in the last 24h? I think with your implementation we will just get both of them but it would be nice to write a test for this